pre-commit.sh

```diff
@@ -105,7 +107,7 @@ function run_integration_tests
     fi
 }
 
-if [[ -z "$PYTHON_VIRTUAL_ENVS" ]]
+if [[ -z "${PYTHON_VIRTUAL_ENVS+1}" ]]
```

Comment:
Looking at StackExchange, I think I like this answer better:

"You can also use `${foobar-}` to get an empty string when foobar is undefined and the value of foobar otherwise (or put any other default value after the -."

According the the bash manual, it should actually be: `${foobar:-}`